FEEDBACK.md

```diff
@@ -0,0 +1,9 @@
+General comments/feedback:
+
+Overall think it looks great. Clear and easy to follow, and plenty of good content. Here's some general feedback. Let me know if any of these sound good and I can make separate PRs for them:
+
+1. Should we include a brief overview of programming in general at the very beginning? High level, ‘what is a program?’ sort of stuff?
+2.  It might be helpful to add an example at the beginning of conditionals that makes the difference between a true and false condition really obvious. For example we could use the window size as the conditional and manually resize the window to show different outcomes
+3. For the rock, paper, scissors, game in practice problems what if we set it up so two people could actually play together? We could setup two window.prompt variables and an empty function and have the students complete the rest from there
```

Comment:
Hmm, there's no need to rewrite as a function tho if the inputs are coming from prompt. let's just do the first one:
```
var playerOnePrompt = prompt('Player 1: Rock, paper, scissors?');
var playerTwoPrompt = prompt('Player 2: Rock, paper, scissors?');

function rockPaperScissors (/Arguments here/) {
/Your code here/
}

rockPaperScissors(/*args?*/)
```